### ISSUE TITLE
Deb install erro on Ubuntu

### ISSUE THREAD

**Issue by glpibr** [jgraph/drawio#1870]
To install on Ubuntu, return erro to search some "LibreOffice" paths
May be an erro on script isntall to find the directory,

An other problem is: the symboic link was not created, but, if you run "/opt/draw.io/drawio" it up.

![image](https://user-images.githubusercontent.com/22614882/110837537-1c784180-8280-11eb-9786-8eeaa246eab3.png)

**Comment by davidjgraph** [comment]
Never seen the libreoffice issues. Try the latest build, the executable has the dot removed.

**Comment by glpibr** [comment]
I tried it now. But the erro is the same. 

May be is a Regular Expression in install script. I dont now!

I made upgrade of LibreOffice and install others programs with DPKG, but no get the same erro with draw.io.

![image](https://user-images.githubusercontent.com/22614882/113710543-39a30300-96ba-11eb-9324-65f57fe53e5d.png)

I made the upgrade draw.io upgrade and the message repeat:

![image](https://user-images.githubusercontent.com/22614882/113710661-60613980-96ba-11eb-841f-35a07f47a233.png)
